models/commons/src/price/price-draft/builder.spec.ts

```diff
@@ -14,8 +14,6 @@ describe('builder', () => {
         value: expect.objectContaining({
           centAmount: expect.any(Number),
           currencyCode: expect.any(String),
-          type: 'centPrecision',
-          fractionDigits: expect.any(Number),
```

Comment:
`Money` inputs in drafts don't contain these two fields. See the [docs](https://docs.commercetools.com/api/types#money)